components/raftstore/src/store/worker/split_check.rs

```diff
@@ -347,5 +347,6 @@ where
         region_epoch,
         split_keys,
         callback: Callback::None,
+        source: "split checker".into(),
```

Comment:
How about pass it as a parameter

Reply:
It's a private function, only used in split checker.

Reply:
Although currently it only used in the split checker, but it is a function not a method of the split checker and nothing can stop people misused it at other places as the function name implying.